xep-0372.xml

```diff
@@ -126,6 +123,48 @@
 </message>
 ]]></example>
   </section2>
+
+  <section2 topic='Anchors' anchor='anchors'>
+    <p>By default it's assumed that the text being referenced is inside the 'body' element of the same stanza in which the 'reference' occurs. There are however situations where the referenced text is inside a different stanza, or inside a different element in the same stanza. In these cases, an 'anchor' attribute is used to clarify which element the 'reference' refers to.</p>
+
+    <section3 topic='Specifying which text a reference refers to' anchor='usecase_other_text'>
+      <p>A stanza might have other text-containing elements besides the 'body', such as the 'subject', 'reason' or 'text' elements. The 'anchor' attribute is used to disambiguate the references in a stanza that has multiple text-containing elements. In this case, the 'anchor' value MUST be a URI fragment which refers to an element in the same stanza (or its value MUST be a full URI with fragment, if the element is in a different stanza). Similarly to HTML, the URI fragment refers to an element with an 'id' attribute that has the same value.</p>
+      <example caption='A reference that refers to text inside the subject, not the body'><![CDATA[
+  <message type='headline'
+          id='a8hef4-99hct232h0'
+          from='verona@channels.shakespeare.lit'>
+    <subject id="subject">Enter SAMPSON and GREGORY, of the house Capulet, armed with swords and bucklers</subject>
+    <body>Gregory, o' my word, we'll not carry coals</body>
+    <reference xmlns='urn:xmpp:reference:0'
+              begin='6'
+              end='13'
+              type='mention'
+              anchor='subject'
```

Comment:
On the mailing list your example used fragment URIs so anchor='#subject' - was the change here deliberate?